.jenkins/actions/run_physics_regression_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e -x
+BACKEND=$1
+EXPNAME=$2
+XML_REPORT="sequential_test_results.xml"
+export TEST_ARGS="-v -s -rsx --backend=${BACKEND} "
+
+# sync the test data
+make get_test_data
+
+if [ ${python_env} == "virtualenv" ]; then
+    export TEST_ARGS="${TEST_ARGS} --junitxml=${jenkins_dir}/${XML_REPORT}"
+    CONTAINER_CMD="srun" make tests physics_savepoint_tests
+else
+    export TEST_ARGS="${TEST_ARGS} --junitxml=/.jenkins/${XML_REPORT}"
+    VOLUMES="-v ${pwd}/.jenkins:/.jenkins" make tests physics_savepoint_tests
```

Comment:
This volume shouldn't need to be mounted to run these make targets, should it? The physics Makefile shouldn't be accessing `../.jenkins` (and doesn't appear to be), and the tests inside the container shouldn't be reading `.jenkins` either.

Reply:
You're right. I'm guessing it was mounted to grab xml report from gce, but it does not actually work. In the current plan, the xml report is always from daint. Removing this.